src/Blumchen/Subscriptions/SubscriptionOptionsBuilder.cs

```diff
@@ -74,10 +73,9 @@ public SubscriptionOptionsBuilder WithReplicationOptions(ReplicationSlotManageme
     }
 
     [UsedImplicitly]
-    public SubscriptionOptionsBuilder Consumes<T, TU>(TU consumer) where T : class
-        where TU : class, IConsumes<T>
+    public SubscriptionOptionsBuilder Consumes<T>(IHandler<T> handler) where T : class
```

Comment:
**COMMENT:** In the long term I'd also like to allow handlers that don't use marker interfaces, but that's something to tackle in the dedicated PR.